.github/workflows/automerge-dependabot.yml

```diff
@@ -0,0 +1,32 @@
+---
+name: Dependabot auto-merge
+on: pull_request
+
+concurrency:
+  group: ${{ github.workflow }}-${{ (github.event.pull_request && github.event.pull_request.number) || github.ref || github.run_id }}
+  cancel-in-progress: true
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    # Changed from the recommended in GH docs "${{ github.actor == 'dependabot[bot]' }}"
```

Comment:
could you share a link? maybe we should make them aware of that.

In https://docs.github.com/en/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions, I only see `if: github.event.pull_request.user.login...`

Reply:
huh, that changed then let me double check.

Reply:
fixed